cpp/src/slice2java/Gen.cpp

```diff
@@ -3570,32 +3568,6 @@ Slice::Gen::TypesVisitor::visitOperation(const OperationPtr& p)
     }
 }
 
-void
-Slice::Gen::TypesVisitor::emitCompactIdHelper(const ClassDefPtr& p)
```

Comment:
Removed these generated helper classes.

As of this PR, if your Java app receives classes with Compact ID,  you need to create a ClassSliceLoader with these classes.